apps/ringtones/js/tone_player.js

```diff
@@ -131,28 +127,4 @@ TonePlayer.prototype = {
       this._playingCallback(playing);
     }
   },
-
-  /**
-   * Creates an AudioContext with "ringer" priority to stop any background audio
-   * from playing once we've started previewing ringtones, or destroys the
-   * context.
-   *
-   * @param {Boolean} exclusive true to create the AudioContext, false to
-   *   destroy.
-   */
-  _setExclusiveMode: function(exclusive) {
-    if (exclusive) {
-      if (!this._source) {
-        this._context = new AudioContext('ringer');
```

Comment:
Should you also remove the 'audio-channel-ringer' permission from the manifest? 

And don't you need to set mozAudioChannelType = 'content' somewhere now that you're taking this out?  Without that won't the ringtone just play over the top of background music? And I suppose then you need audio-channel-content permission?